__tests__/Cars.test.js

```diff
@@ -0,0 +1,39 @@
+import Cars from "../src/class/Cars";
+
+describe("Cars Class 테스트", () => {
+  test("addCar를 통해 이미 존재하는 자동차를 추가하면 에러가 발생한다.", () => {
+    const carsModel = new Cars();
+
+    carsModel.addCars(["test1", "test2"]);
+
+    expect(() => carsModel.addCar("test1")).toThrowError(
+      "자동차 이름은 중복될 수 없습니다.",
+    );
```

Comment:
```suggestion
    carsModel.addCars([new Car("test1"), "test2"]);

    expect(() => carsModel.addCar("test1")).toThrowError(
      "자동차 이름은 중복될 수 없습니다.",
    );
```
```suggestion
    carsModel.addCars(["test1", "test2"]);

    expect(() => carsModel.addCar(new Car("test1"))).toThrowError(
      "자동차 이름은 중복될 수 없습니다.",
    );
```
이런 경우도 고려해보면 좋아요!
Cars에 string이 아니라 아예 Car가 들어오는 경우, 어떻게 처리해주는게 좋을까요?